### PR TITLE
Journey cost

### PR DESCRIPTION
**Pull request by CarlOllington**

### CODE REVIEW

**Review by Henry-Tottle — COMMENTED** [review]
YES

File: functions.js [hunk 1]
```diff
@@ -10,6 +10,24 @@ const getAllStations = async (req, res) => {
     }
 }
 
+const calculateCosts = (originZone, destinationZone) => {
+
+    let cost = 0
```
Comment (matt-ioa) [inline functions.js]: Couldn't you just set the cost to 3.99 in the first place, since you're adding that in all cases? Then you can remove the first branch in the conditional.

File: functions.js [hunk 2]
```diff
@@ -81,7 +100,7 @@ const getJourneys = async (req, res) => {
                 return {
                     line: originStation.line,
                     start: Math.min(originStation.position, destinationStation.position),
-                    end: Math.max(originStation.position, destinationStation.position)
+                    end: Math.max(originStation.position, destinationStation.position),
```
Comment (matt-ioa) [inline functions.js]: Why did you add this comma? It's the last item in the object.

**Review by matt-ioa — CHANGES_REQUESTED** [review]
Just a couple of changes needed

**Review by matt-ioa — APPROVED** [review]
Good job!
